client/core/core.go

```diff
@@ -1087,6 +1087,19 @@ type Config struct {
 	TorIsolation bool
 	// Language. A BCP 47 language tag. Default is en-US.
 	Language string
+
+	// NoAutoWalletLock instructs Core to skip locking the wallet on shutdown.
+	// This can be helpful if the user wants the wallet to remain unlocked. e.g.
+	// They started with the wallet unlocked, or they intend to start Core again
+	// and wish to avoid the cost of unlocking a locked the wallet on startup.
+	NoAutoWalletLock bool // zero value is legacy behavior
+	// NoAutoDBBackup instructs the DB to skip the creation of a backup DB file
+	// on shutdown. This is useful if the consumer is using the BackupDB method,
+	// or simply creating manual backups of the DB file after shutdown.
+	NoAutoDBBackup bool // zero value is legacy behavior
+	// UnlockCoinsOnLogin indicates that on wallet connect during login, or on
+	// creation of a new wallet, all coins with the wallet should be unlocked.
+	UnlockCoinsOnLogin bool
```

Comment:
Latest mole-whack: https://github.com/decred/dcrdex/pull/1849
It'll never be a guaranteed-solved problem IMO, although we'll keep trying.  This is just an option for advanced Core consumers who "know what they're doing" to avoid it.